product-service/serverless.ts

```diff
@@ -63,6 +130,39 @@ const serverlessConfiguration: Serverless = {
             path: '/products',
             cors: true
           }
+        },
+        {
+          sqs: {
+            batchSize: 1,
+            arn: {
+              'Fn::GetAtt': [ 'catalogItemsQueue', 'Arn' ]
+            }
+          }
+        }
+      ]
+    },
+    catalogBatchProcess: {
+      handler: 'handler.catalogBatchProcess',
+      events: [
+        {
+          http: {
+            method: 'post',
+            path: '/catalog-batch-process',
+            cors: true
+          }
+        }
+      ]
+    },
+    createProductTopic: {
```

Comment:
Ok, now I'm lost. Which one of these handlers is supposed to cover points 6.1 and 6.2?

Reply:
Updated, I had a wrong understanding of this task